conda_smithy/templates/build_steps.sh.tmpl

```diff
@@ -58,6 +62,15 @@ if [[ "${BUILD_WITH_CONDA_DEBUG:-0}" == 1 ]]; then
     endgroup "Running conda debug"
     # Drop into an interactive shell
     /bin/bash
+elif [[ "${BUILD_WITH_MAMBABUILD:-0}" == 1 ]]; then
+    startgroup "Running conda mambabuild"
+    if [[ "x${BUILD_OUTPUT_ID:-}" != "x" ]]; then
+        EXTRA_CB_OPTIONS="${EXTRA_CB_OPTIONS:-} --output-id ${BUILD_OUTPUT_ID}"
+    fi
+    conda mambabuild "${RECIPE_ROOT}" -m "${CI_SUPPORT}/${CONFIG}.yaml" \
+        ${EXTRA_CB_OPTIONS:-} \
+        --clobber-file "${CI_SUPPORT}/clobber_${CONFIG}.yaml"
+    endgroup "Running conda mambabuild"
```

Comment:
If this doesn't upload I think it should always mark the job as failing else it's going to get missed.

Reply:
yeah I was also thinking that.

Reply:
Easiest way is to set the option to upload packages to false.